charts/timed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.3.20
+version: 0.3.21
```

Comment:
Because you're introducing a new feature you should bump the minor release.

Reply:
Strictly speaking SemVer only mandates such a bump after reaching the 1.x version. In this case it makes sense to bump minor anyhow ;)